docs/design/coreclr/profiling/davbr-blog-archive/Profiler stack walking Basics and beyond.md

```diff
@@ -145,77 +145,77 @@ But before you get too deep, note that the issue of whether and how to seed a st
 
 For the truly adventurous profiler that is doing an asynchronous, cross-thread, seeded stack walk while filling in the unmanaged holes, here’s what it would look like.
 
-| 
+|
 
-Block of  
-Unmanaged  
+Block of
+Unmanaged
 Frames
```

Comment:
Are these meant to be in separate lines?

If so, they should be:

```suggestion
Block of\
Unmanaged\
Frames
```

Reply:
I think it would be fine for the whole "Block of Unmanaged Frames" to be on the single line, in both the rendered view and raw .md.